pkgs/games/astromenace/default.nix

```diff
@@ -12,27 +12,29 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ cmake ];
   buildInputs = [ xlibsWrapper libGLU libGL SDL openal freealut libogg libvorbis ];
 
-  buildPhase = ''
-    cmake ./
-    make
+  postBuild = ''
     ./AstroMenace --pack --rawdata=../RAW_VFS_DATA
   '';
+
   installPhase = ''
     mkdir -p $out/bin
+
     cp AstroMenace $out
     cp gamedata.vfs $out
+
     cat > $out/bin/AstroMenace << EOF
     #!${runtimeShell}
     $out/AstroMenace --dir=$out
     EOF
+
     chmod 755 $out/bin/AstroMenace
   '';
 
-  meta = {
+  meta = with lib; {
     description = "Hardcore 3D space shooter with spaceship upgrade possibilities";
     homepage = "https://www.viewizard.com/";
-    license = lib.licenses.gpl3;
-    platforms = lib.platforms.linux;
+    license = licenses.gpl3;
```

Comment:
`gpl3` is deprecated. `gpl3Only` or `gpl3Plus`?

Reply:
How can I tell the difference? [Repology can't really decide](https://repology.org/project/astromenace/information#licenses)

Reply:
It's best to look at the source code (https://github.com/viewizard/astromenace/blob/master/LICENSE.md). The source code itself would then be `gpl3Plus`, as it contains the magic words `either version 3 of the License, or (at your option) any later version.`